apps/st2-history/template.html

```diff
@@ -113,7 +113,14 @@ <h2 class="st2-flex-table__caption-title">{{ group.period | date:'fullDate':'UTC
                   {{ record.trigger.type }}
                 </span>
               </span>
-              <span ng-switch-default>Manual</span>
+              <span ng-switch-default>
+                <span class="st2-history__column-app-name">
+                  Manual
```

Comment:
To give you an idea - at cloud monitoring, we allowed user to pass special `_who` and `_why` query parameter with each API call - http://docs.rackspace.com/cm/api/v1.0/cm-devguide/content/general-api-info-audits.html

This parameter can then either be populated manually by the user or automatically via the tool (e.g. CLI / st2web) or a combination of those two.

This is very useful for audit and tracking purposes and we should also do something along those lines as well.